src/editors/EditorProvider.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import Placeholder from '@edx/frontend-lib-content-components';
+
+const EditorProvider = () => (
+  <div className="editor-page">
+    <Placeholder />
```

Comment:
Nit: Could include the course ID here?

Reply:
The linter would yell at me because it would be unused :(.